src/engine/source/base/parseEvent.cpp

```diff
@@ -123,9 +123,9 @@ Event parseOssecEvent(const std::string& event)
                 "Engine base event parse: \"{}\" method: Event received: \"{}\".",
                 __func__,
                 event);
-            throw fmt::format(
+            throw std::runtime_error(fmt::format(
```

Comment:
throwing an exception whose type 'std::string' is not derived from 'std::exception'